lib/src/browser_transport_platform.dart

```diff
@@ -22,23 +22,23 @@ class BrowserTransportPlatform implements TransportPlatform {
 
   /// Construct a [FormRequest] instance for use in the browser.
   @override
-  FormRequest newFormRequest() => new BrowserFormRequest();
+  FormRequest newFormRequest() => new BrowserFormRequest(this);
```

Comment:
In order to support cloning (and auto-retrying), the transport platform needs to be passed into the request class so that it can be passed on to the clone.